libv8.gemspec

```diff
@@ -21,6 +21,7 @@ Gem::Specification.new do |s|
     `git ls-files`.split("\n").reject {|f| f =~ /^out/}.map {|f| "vendor/v8/#{f}"}
   end
   s.files += Dir['vendor/v8/build/**/*']
+  s.files += Dir['vendor/gyp/**/*']
```

Comment:
Without that line I could not compile but i don't remember if it was due to folder presence or to what it contains